### DISCUSSION TITLE
Can't I use env() in vanilla-extract?

### DISCUSSION THREAD

**Discussion by juyeong-s** [vanilla-extract-css/vanilla-extract#1447] [Q&A]
I applied env(safe-area-inset-bottom, 16px) to paddingBottom as follows, but it is not applied. Are there any other instructions for using it?

```ts
export const button = style({
  position: 'fixed',
  right: 0,
  bottom: 0,
  left: 0,
  width: '100%,
  height: '56px',
  fontWeight: '700',
  paddingBottom: 'env(safe-area-inset-bottom, 16px)',
});
```

**Comment by askoufis** [comment]
Nope, it should work just fine. Your `style` looks fine (though you're missing a `'` at the end of `width: '100%`, but I'm guessing that's just a bad copy/paste). Do you have a minimal reproduction that I could look at?
